### DISCUSSION TITLE
How to deploy client to subdirectory? https://subdomain.domain.com/subdirectory

### DISCUSSION THREAD

**Discussion by cc8t** [WongSaang/chatgpt-ui#223] [Q&A]
Reason:
I have a common domain: https://subdomain.domain.com/subdirectory, and each subdirectory is a reverse proxy of another app server machine across my on premise office. I've tried ask GPT3.5 for help but unable to do so

if react can refer like this: https://skryvets.com/blog/2018/09/20/an-elegant-solution-of-deploying-react-app-into-a-subdirectory/

but for nuxt3, I did try add /subdirectory/ to app.baseUrl and it work to serve the pages, however the django APIs(/api/**) messed up, any pointers?

**Comment by cc8t** [comment]
<img width="922" alt="image" src="https://github.com/WongSaang/chatgpt-ui/assets/72854949/87209bac-8f4f-4b27-b871-a6321b28cb2e">

**Comment by spencerwongfeilong** [comment]
Why not deploy to subdirectory? Isn't it easier.? 

For example chatgpt.domain.com

**Reply by cc8t** [reply]
That is not applicable as the main machine having more than 20 VM, so one of the VM is at least 1 of the subdirectory

**Reply by spencerwongfeilong** [reply]
Sorry not sure how to help you there.
